content/docs/concepts/security/index.md

```diff
@@ -239,6 +271,41 @@ Istio provides two types of authentication:
   original client making the request as an end-user or device. Istio
   supports authentication with JSON Web Token (JWT) validation.
 
+### Mutual TLS authentication
+
+Service-to-service communication is tunneled through the client side [Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy.
```

Comment:
To avoid passive voice, I suggest:
Istio tunnels service-to-service communication through the client side and server side [Envoy](https://envoyproxy.github.io/envoy/) proxies.